content/200-concepts/100-components/02-prisma-client/062-computed-fields.mdx

```diff
@@ -80,21 +80,20 @@ generator client {
 }
 
 model User {
-  id    Int     @id @default(autoincrement())
-  email String  @unique
-  name  String?
-
-  posts Post[]
+  id         Int     @id @default(cuid())
+  email      String  @unique
+  firstName  String
+  lastName   String
+  posts      Post[]
 }
 
 model Post {
-  id        Int     @id @default(autoincrement())
-  title     String
-  published Boolean @default(true)
-  content   String?
-
-  authorId Int?
-  author   User? @relation(fields: [authorId], references: [id])
+  id         Int     @id @default(cuid())
```

Comment:
```suggestion
  id         Int     @id @default(autoincrement())
```

Reply:
@ruheni I updated `cuid()` instead of `autoincrement()` follow example data here https://github.com/prisma/docs/pull/4955/files/b5457fba7ef3292949be3399d8952aec2287a5c0#diff-706e32490d273cb9040323cf405bd6cd87e12b15175b265afb9b457117b36445R56.

Reply:
Good catch! In that case, let’s update the id field in the response object to an integer